apispec/LazyDict.py

```diff
@@ -0,0 +1,36 @@
+class LazyDict(dict):
+
+    def values(self):
+        return [self.__getitem__(key) for key in self.keys()]
+
+    def items(self):
+        return [(key, self.__getitem__(key)) for key in self.keys()]
+
+    def iteritems(self):
+        return [(key, self.__getitem__(key)) for key in self.keys()]
+
+    def itervalues(self):
+        return [self.__getitem__(key) for key in self.keys()]
+
+    def get(self, k, d=None):
+        return self.__getitem__(k) if k in self else d
+
+    def __init__(self, *args, **kwargs):
+        super(LazyDict, self).__init__()
+        self.update(dict(*args, **kwargs))  # use the free update to set keys
```

Comment:
yes I guess it was a remnant of another experiment, I guess I can remove the `__init__` entirely